application/controllers/admin/globalsettings.php

```diff
@@ -480,6 +480,65 @@ public function surveysettingmenues()
         Yii::app()->getController()->renderPartial('super/_renderJson', ['data' => $menues[0]]);
     }
 
+    public function sendTestEmail()
+    {
+        $sSiteName = Yii::app()->getConfig('sitename');
+
+        //Use the current user details for the default administrator name and email
+        $user = User::model()->findByPk(Yii::app()->session['loginID']);
+        $sTo = $user->full_name . " <" . $user->email . ">";
+        $sFrom = Yii::app()->getConfig("siteadminname") . " <" . Yii::app()->getConfig("siteadminemail") . ">";
+        $sSubject = sprintf(gT('Test email from %s'), $sSiteName);
+        
+        $body   = array();
+        $body[] = sprintf(gT('This is a test email from %s'), $sSiteName);
+        $body   = implode("\n", $body);
+
+        $this->_sendEmailAndShowResult($body, $sSubject, $sTo, $sFrom);
+    }
+
+    private function _sendEmailAndShowResult($body, $sSubject, $sTo, $sFrom)
+    {
+        $mailer = new \LimeMailer();
```

Comment:
Better to have https://github.com/LimeSurvey/LimeSurvey/blob/91375a58e22334af287e652885599142954a676f/application/core/LimeMailer.php#L49

Reply:
What do you mean?

Reply:
You don't set it, then it stay default … https://manual.limesurvey.org/BeforeEmail

Reply:
Updated

Reply:
:+1: